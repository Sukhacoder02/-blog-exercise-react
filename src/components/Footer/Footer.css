footer {
  flex: none;
  height: 48px;
  background-color: black;
  width: 100vw;
  display: flex;
  align-items: center;
  justify-content: center;
  /* position: fixed; */
  /* bottom: 0; */
}
footer > .footerHeading > h4 {
  color: white;
}
footer > .footerHeading {
  width: 80%;
  display: flex;
  align-items: flex-start;
  flex-direction: column;
  justify-content: center;
}
