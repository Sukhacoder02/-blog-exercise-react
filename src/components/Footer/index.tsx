import * as React from 'react';
import './Footer.css';

const Footer: React.FunctionComponent = (): JSX.Element => {
  return (
    <footer>
      <div className="footerHeading">
        <h4>@artifact.com 2023</h4>
      </div>
    </footer>
  );
};

export default Footer;
