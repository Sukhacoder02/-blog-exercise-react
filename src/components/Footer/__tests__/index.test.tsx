import * as React from 'react';
import { render } from '@testing-library/react';
import Footer from '..';

describe('Component Footer', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
