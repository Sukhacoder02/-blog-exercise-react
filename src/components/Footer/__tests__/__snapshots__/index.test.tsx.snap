// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Footer renders without crashing 1`] = `
<DocumentFragment>
  <footer>
    <div
      class="footerHeading"
    >
      <h4>
        @artifact.com 2023
      </h4>
    </div>
  </footer>
</DocumentFragment>
`;
